### DISCUSSION TITLE
Add Linux distributions support

### DISCUSSION THREAD

**Discussion by Chevek** [moraroy/NonSteamLaunchers-On-Steam-Deck#56] [Ideas]
Hi,

Do you have any plan to add support for Linux Distributions (Fedora Linux, Ubuntu, Mint, Debian...) ? Will you accept PR for this ?

I've tested quickly on Fedora Linux 38 installing battle.net and it can work with a minor tweak.

Best regards,
Chevek

**Comment by moraroy** [comment]
@Chevek I'm very open to pull requests and making this script work for as many people as possible feel free to do a pull request! I like the idea of adding more flavors of linux!

**Comment by sigzegv** [comment]
I see two issues.
First, path to ~/Downloads should not be used directly and you should always prefer using $XDG_DOWNLOAD_DIR if present (or /tmp).
Second, `$HOME/.local/share/Steam` should not be hardcoded on everypath, put this in a var and use this var in your paths. (Moreover, in some systems, this path doesn't exists and steam is directly installed in ~/.steam.)

**Reply by moraroy** [reply]
@sigzegv do a pull request and I'd be happy to add it in to main, as long as you can confirm it works.
